ros_buildfarm/templates/snippet/publisher_publish-over-ssh.xml.em

```diff
@@ -13,7 +13,7 @@
                 <removePrefix>@remove_prefix</removePrefix>
                 <remoteDirectorySDF>false</remoteDirectorySDF>
                 <flatten>false</flatten>
-                <cleanRemote>false</cleanRemote>
+                <cleanRemote>@clean_remote</cleanRemote>
```

Comment:
To avoid having to explicitly pass `clean_remote='false'` in all cases where this snippet is used you can check if the snippet was invoked with the argument and fall back to `false`:

```
@(vars().get('clean_remote', 'false'))
```

That will also ensure to not break users having custom templates using the snippet without the new argument.

Reply:
Thank you! That's a really good point.
I changed it accordingly.